qdrant-landing/content/articles/agentic-rag.md

```diff
@@ -8,6 +8,8 @@ weight: -150
 author: Kacper Łukawski
 author_link: https://www.kacperlukawski.com
 date: 2024-11-22T00:00:00.000Z
+url: /articles/embed-data/agentic-rag
```

Comment:
We can not change the URLs of articles; please filter articles by category (the same way as docs are filtered by `partition`)

Reply:
Done